misc/pkg_install.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+LIB_7="vim build-essential ccache libgc-dev graphviz libunwind7-dev pkg-config"
+LIB_8="vim build-essential ccache libgc-dev graphviz libunwind8-dev pkg-config"
+MSG="The following package will be installed [ vim | build-essential | ccache | libgc-dev | graphviz | libunwind-dev | pkg-config ]"
+MSG_VIM="Do you want to set your vim for NIT ? [y/n] "
```

Comment:
if n is the default, make it uppercase. eq `[y/N]`
Shouldn't yes be the default by the way?